app/styles/app.scss

```diff
@@ -82,12 +106,23 @@ body {
   width: 100%;
 }
 
+#footer {
+  padding-top: 50px;
+  a{
+    color:black;
+  }
+  center{
+    color:white;
+    background-color: black;
+  }
+}
 .colorfix {
   background-color: $blue;
 }
 
 .content {
   background-color: $white;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.